languages/javalang/builder-renderer/generated-code/src/test/java/com/commercetools/api_client/ApiClientIntegrationTests.java

```diff
@@ -0,0 +1,47 @@
+package com.commercetools.api_client;
+
+import com.commercetools.client.ApiRoot;
+import com.commercetools.models.ApiClient.ApiClient;
+import com.commercetools.models.ApiClient.ApiClientPagedQueryResponse;
+import com.commercetools.utils.CommercetoolsTestUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ApiClientIntegrationTests {
+    
+    @Test
+    public void createAndDeleteById() {
+        ApiClient apiClient = ApiClientFixtures.createApiClient();
+        ApiClient deletedApiClient = ApiClientFixtures.deleteApiClient(apiClient.getId());
+
+        Assertions.assertEquals(apiClient.getId(), deletedApiClient.getId());
```

Comment:
Here I would personally prefer to have a static import, but we can keep it as it is for now.